packages/azure-http-specs/specs/azure/resource-manager/resources/mockapi.ts

```diff
@@ -98,8 +110,8 @@ const validResourceGroupExtensionsResource = {
 };
 
 const validSubscriptionExtensionsResource = {
-  id: `/subscriptions/${SUBSCRIPTION_ID_EXPECTED}/providers/Azure.ResourceManager.Resources/extensionsResources/extension`,
-  name: "extension",
+  id: `${SUBSCRIPTION_SCOPE_URI}/providers/Azure.ResourceManager.Resources/extensionsResources/extension`,
+  name: `${EXTENSION_RESOURCE_NAME}`,
```

Comment:
Any difference from this?
```suggestion
  name: EXTENSION_RESOURCE_NAME,
```

Reply:
Fixed by the [commit#e90e90c](https://github.com/Azure/typespec-azure/pull/2203/commits/e90e90cf288973e618b4d5cd3029856bc0155cf5)